vue3-apollo-quasar/src/components/PaginationButtons/PaginationButtons.vue

```diff
@@ -0,0 +1,61 @@
+<template>
```

Comment:
Quasar already has [a pagination component](https://quasar.dev/vue-components/pagination) (with the functionality you're implementing here + more) that could be modified to look the same.

Since this PR is not about pagination, we could fix this in a separate PR? @dustinsgoodman , looking at you for clarification on the same 😄